client/my-sites/media-library/header.jsx

```diff
@@ -119,6 +120,20 @@ export default React.createClass( {
 		} );
 	},
 
+	renderUploadAndEditButton() {
+		if ( ! config.isEnabled( 'image-editor' ) ) {
+			return;
+		}
+
+		return (
+			<button className="button is-desktop"
```

Comment:
Didn't exist at the time this component was originally written, but we now suggest using the [`<Button />` component](https://github.com/Automattic/wp-calypso/tree/master/client/components/button) for most buttons in lieu of rendering the base `button` element with classes specified.

What is the plan for editing images on mobile devices?